src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -250,6 +250,10 @@ private static Binder GetEnclosingBinderInternalWithinRoot(SyntaxNode node, int
                 {
                     binder = rootBinder.GetBinder(current);
                 }
+                else if (kind == SyntaxKind.SwitchExpression)
```

Comment:
>else if (kind == SyntaxKind.SwitchExpression) [](start = 16, length = 45)

Is there an associated test?

Reply:
No, there is an open issue for SemanticModel implementation and tests. This is present to unblock an exception in the IDE when hand-testing.

---
In reply to: [179658390](https://github.com/dotnet/roslyn/pull/25963#discussion_r179658390) [](ancestors = 179658390)